hpycc/save.py

```diff
@@ -116,14 +117,17 @@ def save_outputs(connection, script, directory=".", filenames=None,
     if prefix:
         chosen_filenames = [prefix + name for name in chosen_filenames]
 
-    for name, result in zip(chosen_filenames, results.items()):
-        path = os.path.join(directory, name)
+    paths = [os.path.join(directory, name) for name in chosen_filenames]
+    for path, result in zip(paths, results.items()):
```

Comment:
should we make filenames compulsory then?

Reply:
As a user I would actually prefer to use the named results in ECL, it means I know exactly what I'm getting. As such I'd prefer this wasn't there than enforced.

Reply:
I kind of agree with that. Should the user just specify the parent directory then? maybe we should add some flag to do something if the file already exists?

Reply:
Done. Flag for overwrite is True by default. As would any other function if used without care.